plugin/mqtt.go

```diff
@@ -112,6 +112,36 @@ func (m *Mqtt) newReceiver() (*msgHandler, error) {
 	return h, err
 }
 
+// FormatPayload processes a payload and returns what to publish to the topic.
+func (m *Mqtt) FormatPayload(playload_to_format string, param string, v any) (string, error) {
+	payload, err := setFormattedValue(playload_to_format, param, v)
+	if err != nil {
+		return "", err
+	}
+
+	if m.pipeline != nil {
```

Comment:
Wäre es nicht einfacher und weniger diff gewesen, diesen Schritt in setFormattedValue mit einzubauen? Es muss ohnehin auch analog für http, script etc erfolgen.